.github/workflows/publish-prod.yaml

```diff
@@ -3,7 +3,7 @@ name: publish-prod
 on:
   push:
     tags:
-      - "[0-9]+.[0-9]+.[0-9]+"
+      - "0.25.[0-9]+"
```

Comment:
I think this is the right way to do this? Trigger a 0.25 production build only when we create a 0.25.X release.

🚨 We'll have to make sure we update `main` to exclude the archived tags before we publish a 0.25.x release! Otherwise I think `main` will trigger a new prod publish, and deploy the 0.25 site to docs.camunda.io!!! (I think that's what would happen?)

Reply:
PR to prevent accidental 0.25 deployment to docs.camunda.io is up, at https://github.com/camunda/camunda-platform-docs/pull/2172